src/pages/LandingPage/index.tsx

```diff
@@ -35,7 +35,7 @@ const LandingPage = () => {
       <MembersSection>
         <MembersSectionHeader>Groups!</MembersSectionHeader>
         <InformationCardSection>
-          <GroupCard groupNumber={0} devs="name1, name2, name4" designers="name1, name2, name4" />
+          <GroupCard groupNumber={4} devs="Elva, Zhikai, Weipin, Marcus" designers="Gregory, Shannon, Brendan" />
```

Comment:
create your own card instead of replacing existing

Reply:
Fixed in commit 8ef09c6c925d830904433c90327e20ba02ddc34c